### DISCUSSION TITLE
Meat Pack Support - Marlin.

### DISCUSSION THREAD

**Discussion by rorywquin** [Jyers/Marlin#106] [Q&A] [answered]
Hi

Do the precompiled files have MeatPack Support from Marlin?

Tx.

**Comment by Xploder** [comment, marked as answer]
It's being discussed in this topic: https://github.com/Jyers/Marlin/discussions/29

**Reply by rorywquin** [reply]
Thanks - missed that as it started as a different topic. Maybe MP should have it's own discussion?

**Reply by darco415** [reply]
So I see meatpack support added to 1.2.2 but when I view the config files, I dont see any reference to meatpack?
Also a meatpack bugfix was pushed upstream #21306

**Reply by t3chguy** [reply]
https://github.com/Jyers/Marlin/blob/Ender_3_V2_Extensible_UI/Configuration%20Files/E3V2%20Templates/Default-NoProbe/Configuration_adv.h#L3310

**Reply by darco415** [reply]
Oh I see, I was looking at the non template files
